database/README.md

```diff
@@ -51,27 +51,25 @@ The CRIS data in our database consists of four record types:
 
 #### Design
 
-The core challenge that the database schema solves is to preserve the integrity of staff members' edits while simultaneously allowing crash record updates to flow into the database from CRIS. The editing environment is achieved by managing multiple copies of each of the `crashes`, `units`, and `people` tables, so that CRIS edits and Vision Zero staff edits remain isolated.
+The core challenge that the database schema solves is to preserve the integrity of staff members' edits while simultaneously allowing crash record updates to flow into the database from CRIS. The editing environment is achieved by managing two copies of each of the `crashes`, `units`, and `people` tables, so that CRIS edits can have their own table and thus be isolated from Vision Zero staff edits.
 
-For example, the `crashes` records are managed in three tables:
+For example, the `crashes` records are managed in two tables:
 
 - `crashes_cris`: records that are created and updated by TxDOT CRIS through the [CRIS import ETL](../etl/cris_import/README.md)
-- `crashes_edits`: record edits created by Vision Zero staff through the Vision Zero Editor web app
-- `crashes`: a unified version of each record which combines the values in `crashes_cris` plus any values in `crashes_edits`
+- `crashes`: a unified version of each record which combines the values in `crashes_cris` plus any edits created by Vision zero staff through the Vision Zero Editor web app. This table functions as the final source of truth, where VZ edits take precedence over CRIS values.`
```

Comment:
here and further below, i think "user edits" might be more clear than "VZ edits" 🤷